src/internal/net/Client.js

```diff
@@ -75,7 +77,17 @@ export default class Client {
 
   /** @private */
   async authRequest (method, endpoint, payload, headers) {
+    let t0 = performance.now();
+    let statekey = Math.random(100, 10000)
```

Comment:
```suggestion
    let statekey = Math.random().toString(36).substr(2)
```

Just makes this a bit more random...